lib/handler.js

```diff
@@ -74,8 +79,11 @@ Handler.prototype.checkRequestAuthorization = async function checkRequestAuthori
     // when an existing token exists try to authenticate the session
     if (existingToken) {
       await this.saveData({ accessToken: existingToken })
+    } else {
+      errorLog("nuxt-oauth - existing token not found")
```

Comment:
do you think we need to log this as an error? do you think anonymous users will also be hitting this method?